sw/host/opentitanlib/src/util/usr_access.rs

```diff
@@ -107,3 +108,13 @@ pub fn usr_access_set(bitstream: &mut [u8], val: u32) -> Result<()> {
     remove_crc(bitstream);
     Ok(())
 }
+
+/// Returns the crc32 hash of the bitstream to be used as a USR_ACCESS value
+pub fn usr_access_crc32(bitstream: &mut [u8]) -> Result<u32> {
+    usr_access_set(bitstream, 0)?; // Clear usr_access before hashing
+
+    let crc = Crc::<u32>::new(&crc::CRC_32_ISO_HDLC);
+    let mut digest = crc.digest();
+    digest.update(bitstream);
+    Ok(digest.finalize())
```

Comment:
Nit: could eliminate three lines with the `checksum` convenience function. Maybe just inline this function into `set_usr_access` at that point :)

Reply:
Ah I missed that function! Thanks!